Tasks/AzureFileCopyV1/task.loc.json

```diff
@@ -13,7 +13,7 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 1,
-    "Minor": 225,
+    "Minor": 226,
```

Comment:
current sprint is 225

Reply:
Cut-off day is passed, so it should be 226
[guide](https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/taskversionbumping.md)